aws/eks/cloudwatch_log.tf

```
@@ -33,3 +33,15 @@ resource "aws_cloudwatch_log_metric_filter" "celery-error" {
     value     = "1"
   }
 }
+
+resource "aws_cloudwatch_log_metric_filter" "malware-detected" {
+  name           = "malware-detected"
+  pattern        = "Malicious content detected"
```

```suggestion
  pattern        = "Malicious content detected! Download and attachment failed"
```

Could we target this to either just the celery log or just the dd-api log? I think this pattern will match on both logs and then we will be double counting. My suggested code just matches the log from Celery.

---

Sure! I wasn't sure how to target the logs (it just seemed to be looking at the entire EKS log group).  In my clickops testing in staging it didn't double count it, but I like your idea of making it more explicit.  I will make this update!

---

Done.